### ISSUE TITLE
DeclarationOrder linter doesn't work inside media queries

### ISSUE THREAD

**Issue by mrpavlikov** [brigade/scss-lint#423]
Not sure if its a bug or not, but it seems to be. Following code reasonably fails:

```scss
@mixin mix1() {
    color: #ccc;
}

@mixin mix2() {
    border: #ccc;
}

.b-map {
    @include mix1();

    width: 100%;
    height: 100%;

    @include mix2();
}
```

But if I wrap `.b-map` contents into media query lint passes:

```scss
@mixin mix1() {
    color: #ccc;
}

@mixin mix2() {
    border: #ccc;
}

.b-map {
    @media screen and (max-width: 600px) {
        @include mix1();

        width: 100%;
        height: 100%;

        @include mix2();
    }
}
```

**Comment by aurfon** [comment]
+1

**Comment by sds** [comment]
Thanks for the bug report @mrpavlikov.

Fixed in 6fb318effe4e10a6f67f8f7e3914b4519f414720.
